### DISCUSSION TITLE
TLS Cert/Key Return to Default

### DISCUSSION THREAD

**Discussion by seitzy38** [filebrowser/filebrowser#2056] [Q&A]
Forgive me here, I am very very new to any and all of this: 

I've created a filebrowser hosted on my PC and have a domain name for it that came with an SSL Key and Cert. I've correctly put the path on both in Command Line, but when I run Filebrowser, I get several errors; "Client Sent an HTTP request to an HTTPS server" or when I run it to listen on 127.0.0.1 I get an SSL Cert issue. 

I don't need an SSL Cert for this project, it was running just fine before I linked the PEM files, but now I can't return the TLS Key/Cert to the default, which was nothing. Is there anything I can do to return this to default? Just the TLS Key/Cert?

**Comment by ramiresviana** [comment]
>  I get several errors; "Client Sent an HTTP request to an HTTPS server" or when I run it to listen on 127.0.0.1 I get an SSL Cert issue.

Using HTTP to access a File Browser instance that is configured with a TLS key, will show that message. HTTPS works with domain names, accessing a site via ip (like 127.0.0.1) will show certificate errors.

> Is there anything I can do to return this to default? Just the TLS Key/Cert?

You can use CLI to change your TLS options, providing an empty string will disable the functionality. Check https://filebrowser.org/cli/filebrowser-config-set
